metafacture-io/src/main/java/org/metafacture/io/HttpOpener.java

```diff
@@ -51,18 +54,55 @@ public final class HttpOpener extends DefaultObjectPipe<String, ObjectReceiver<R
 
     private static final String ACCEPT_HEADER = "accept";
     private static final String ENCODING_HEADER = "accept-charset";
-
```

Comment:
I'm fine with giving up the grouping, but then the constants should be sorted IMO.